testing/README.md

```diff
@@ -30,7 +31,7 @@
 * We encourage using the `screen` object exported by Testing Library because the resulting idioms will be more similar in React and Vanilla JS projects. [1](https://testing-library.com/docs/queries/about#screen), [2](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen).
 
 ## Favorite Tools
-* Unit testing: Jest + Testing Library
+* Unit/integration testing: Jest + Testing Library
```

Comment:
While I don't want us to get hung up on what is the difference between a unit test versus an integration test, I also don't want to _limit_ the types of tests we write. 

Allowing myself to be okay with having the flexibility to maybe begin with a unit test and have it (or a second test file) bleed into an integration test has also lifted a weight off of my "getting more comfortable with writing tests" journey. 🙂

Reply:
I agree with what you said, but Jest is not a good tool for integration tests. So if the goal is to mention "integration test" so that the doc acknowledges the merit of those types of test, maybe just add a new bullet point in this section...

> Integration testing: TBD

I don't believe we have a "favorite tool" for those yet. And I don't want to list one here arbitrarily. So I'm find with just leaving this as a placeholder.

Reply:
Do you define the difference between unit and integration tests as "test an individual piece" vs "test the whole system"? If that is the definition you use, then I would say that Jest is just fine for integration tests. In general, I don't think it is any worse suited for testing whole systems than it is for testing individual pieces.

Reply:
Thanks, @calebeby and @emersonthis. I was going to follow-up and say that Jest has worked great for me when writing both unit and integration tests.

In my mind, unit tests test individual units (a utility function, for example, smallest units of code), while an integration test combines multiple units (or modules) together. 

On the client project I'm on now, I had a more complex UI component that was composed of smaller simpler components. I opted to not write unit tests for each smaller component and instead wrote one integration test that gave me the confidence that the smaller components worked as designed, when integrated, as a larger unit/component. In this case, the integration test provided me the confidence I was looking for while striking a balance between having no tests at all versus writing too many tests with little or no return value. 🙂

Reply:
That probably wouldn't have been my definition, but that doesn't make it not correct. If that's what Gerardo means, then I agree Jest should be fine. Maybe that ambiguity is the issue here. I agree with Gerardo that it's not useful to fret over what is/isn't a certain kind of test. So maybe we could say something like: 

> In ____ situation, we use ____ tool. 

There's probably a better version of this, but the idea is to sidestep the labels and just speak to the strengths of the tools. Maybe?